src/test/java/com/unzer/payment/business/AbstractPaymentTest.java

```diff
@@ -197,13 +197,9 @@ protected Charge getCharge(String typeId, String customerId, String orderId, Str
         return charge;
     }
 
-    protected Card createPaymentTypeCard() throws HttpCommunicationException {
-        return createPaymentTypeCard("4711100000000000");
-    }
-
-    protected Card createPaymentTypeCard(String cardnumber) throws HttpCommunicationException {
+    protected Card createPaymentTypeCard(Unzer unzer, String cardnumber) throws HttpCommunicationException {
```

Comment:
suggestion: Isn't it more work to set the card number manually every time? Couldn't you just add another method like that allows you to set a specific card number if needed?